### ISSUE TITLE
Meeting Reactions not working with german Teams client

### ISSUE THREAD

**Issue by Callidus2000** [tdalon/ahk#10]
Hi,
is it possible to get Teamsy working with the german Teams Client?

**Comment by tdalon** [comment]
Hi and thanks for your interest in my repo :-)
Teamsy and the whole repo were only implemented for an English language setting. (The one I use) It is possible to support other languages but requires some development effort. And till now you are the only one who has requested this.

Are you only talking about meeting reactions or in general? Please be precise in your request.

Some features like Meeting reactions are based on findtext/ imagesearch. Language shall not be an issue here. But anyway it might be necessary to take your screenshot with FindText to get it work for you.

Some features are based on the native hotkeys assigned in Teams. (not sure if hotkeys are lang. neutral) -

Some other Teamsy features are based on some Teams [commands](https://support.microsoft.com/en-us/office/use-commands-in-teams-88f61508-284d-417f-a53d-9e082164050b) like /call: here I remember that the commands are definitely language specific. 

If you are precise about what is not working for you and what would be the fix (e.g. list the commands in your language), I can have a look if I can implement this quickly.
In general, you are free to edit the code to your specific needs.
If it isn't something that I can do quickly and you would really like it, you might convince me to implement it for you by offering me some beer ;-)

**Comment by itb-devs-de** [comment]
This is an issue for me too. Almost none of the features are working because they rely on UI text instead of internal control names. For example the Mute/Unmute is broken because the Name is "Stummschalten" instead of "Mute".

Generally for UI automation it is better to not rely on UI text but on control names. For example the Mute button has the AutomationId "microphone-button", which can easily be targeted for all language clients equally.

![image](https://user-images.githubusercontent.com/31128213/233578599-3a438500-fdfa-4c98-8a6a-9aa7e0b04339.png)

**Comment by tdalon** [comment]
I have on my backlog to offer a setting to specific the client language used - but still you will have to implement your specific language on your own.

@itb-devs-de : I use the automation-id where it is possible. If I have hardcoded the Mute name this is for the feature to detect the mute state.
Unfortunately the last time I have checked, the Teams client does not offer an automation-id for meeting reactions Therefore the implementation is language specific :-( 
I think I have even reported this to Microsoft but it is a bit hopeless here that MS improves the way the Teams client is implemented.

You are free to make any code improvement with a pull request if you know a better way to solve this issue.
